changes/bug24953

```diff
@@ -0,0 +1,3 @@
+  o Minor bugfixes (fallback scripts):
+    - In updateFallbackDirs.py, say "fallback list" instead of "whitelist".
+      Fixes bug 24953; bugfix on 0.3.5.1-alpha.
```

Comment:
check_existing mode was added in commit 3fd58aa98dbae47ecb4a171f71278a78cf1e488e in 0.3.0.3-alpha:
```suggestion
      Fixes bug 24953; bugfix on 0.3.0.3-alpha.
```

Reply:
Fixed in bf749490432c4e82fc415434fa9f380225303b71.